common/device-manager.ts

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
q: curious why this is named `common/` is it used by something else than the tests?

Reply:
Not right now, but the intent was to have the chrome implementation use these stubs as well

Reply:
Maybe consider re-organizing the codebase when start introducing the chrome implementation?

Reply:
Renamed to `platform`